src/models/document/document-types.ts

```diff
@@ -37,6 +37,9 @@ export function isPublishedType(type: string) {
   return [ProblemPublication, PersonalPublication, LearningLogPublication, SupportPublication]
           .indexOf(type) >= 0;
 }
+export function isSortableType(type: string){
```

Comment:
@scytacki just pointing out that I added this for the filter - not sure it's the optimal way, but hopefully well enough for this stage.